pkg/metrics/grafana/tiproxy_summary.json

```diff
@@ -263,7 +263,7 @@
                      "expr": "tiproxy_server_connections{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}",
                      "format": "time_series",
                      "intervalFactor": 2,
-                     "legendFormat": "{{instance}}",
+                     "legendFormat": "{{instance}} | {{addr}}",
```

Comment:
I don't think we need to adapt to that: Gateway doesn't use TiProxy metrics.

Reply:
Oops, I just saw the comment https://github.com/pingcap/tiproxy/pull/393#issuecomment-1788751694